packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.css

```diff
@@ -0,0 +1,42 @@
+.searchbox__container{
+    display: flex;
+    position: relative;
+    border-color: #e5e8e8;
+    height: 48px;
+    box-sizing: border-box;
+    background: #FFFFFF;
```

Comment:
Maybe we can benefit from some css variable for the color and magic lenght ?

Reply:
What do you think, should we share css between the searchboxes?
https://help.salesforce.com/s/articleView?id=release-notes.rn_lwc_css_share.htm&type=5&release=226

Reply:
Oh yes!